### PR TITLE
docs(clients): Add multi-tenancy to Dgraph client docs

### PR DESCRIPTION
**Pull request by bucanero**
Fixes DOC-183

### CODE REVIEW

File: content/clients/java.md [hunk 1]
```diff
@@ -61,6 +61,24 @@ DgraphStub stub3 = DgraphGrpc.newStub(channel3);
 
 DgraphClient dgraphClient = new DgraphClient(stub1, stub2, stub3);
 ```
+#### Login using ACL
```
Comment (abhimanyusinghgaur) [inline content/clients/java.md:64]: The same section is also present here in this same file from the old README: https://github.com/dgraph-io/dgraph-docs/pull/89/files#diff-547178edb58c1a6cabd82fd993e8b0eef16946db91430386f46e5697c881a268R139-R143

Maybe we merge these both, keep only one of them? Put this one in place of the old section?

Comment (bucanero) [reply content/clients/java.md]: oh, good catch. I removed the duplicate. 👍

File: content/clients/go.md [hunk 2]
```diff
@@ -65,6 +65,29 @@ func newClient() *dgo.Dgraph {
...
+dc := dgo.NewDgraphClient(api.NewDgraphClient(conn))
+ctx := context.Background()
+// Login to namespace 123
+if err := dc.LoginIntoNamespace(ctx, "groot", "password", 123); err != nil {
+	glog.Error("Failed to login: ",err)
+}
+```
+
+In the example above, the client logs into namespace `123` using username `groot` and password `password`.
+This client will be able to access all the data which is accessible to the `groot` user in the namespace `123`.
```
Comment (ahsanbarkati) [inline content/clients/go.md:89]: Once logged in, the client can be used to perform all the operations allowed to the `groot` user of namespace `123`.

Comment (bucanero) [reply content/clients/go.md]: fixed

**Review by anurags92 — APPROVED** [review]

**Pull request merged**